packager/certcache/certcache.go

```diff
@@ -256,9 +385,23 @@ func (this *CertCache) readOCSP() ([]byte, time.Time, error) {
 		this.ocspUpdateAfter = ocspUpdateAfter
 	}
 	return ocsp, ocspUpdateAfter, nil
+
+}
+
+// Print # of retries, wait for specified time and returned updated wait time.
+func waitForSpecifiedTime(waitTimeInMinutes int, numRetries int) int {
+	log.Printf("Retrying OCSP server: retry #%d\n", numRetries)
+	// Wait using exponential backoff.
+	log.Printf("Waiting for %d minute(s)\n", waitTimeInMinutes)
+	waitTimeDuration := time.Duration(waitTimeInMinutes) * time.Minute
+	// For exponential backoff.
```

Comment:
There is no "right" read pattern, so feel free to ignore as bikeshedding.

I think that the cost of making an OCSP request to the CA is reasonably low enough that we could do it every minute or 10 until it succeeds or we time out.

With this backoff, the last lookup is 8hrs apart? Seems excessively long to wait. I'd consider something like every 30s for 5 minutes, and then every 5 minutes for the next 12 hrs.

Reply:
I changed the logic so that it caps the wait time at 10 minutes, max.